src/main/java/de/geolykt/enchantments_plus/util/Utilities.java

```diff
@@ -222,11 +222,12 @@ public static Location getCenter(Location loc, boolean centerVertical) {
         if (centerVertical) {
             y = (int) y + .5;
         }
-        if (z >= 0) {
-            z += .5;
-        } else {
-            z += .5;
-        }
+//        if (z >= 0) {
+//            z += .5;
+//        } else {
+//            z += .5;
+//        }
+        z += .5;
```

Comment:
smells like a bug. You should've corrected it or at least inserted a FIXME in it